### ISSUE TITLE
Issue with training dataset

### ISSUE THREAD

**Issue by depakaakula** [oarriaga/face_classification#43]
Do I have to let it train 10000 times really? 

<img width="840" alt="screen shot 2017-11-19 at 13 56 22" src="https://user-images.githubusercontent.com/32492052/32994171-9f9a3422-cd31-11e7-8dbf-80a5ba3b37f5.png">

**Comment by oarriaga** [comment]
No, the model checkpoint will stop saving the models whenever the validation loss function stops decreasing.

**Comment by depakaakula** [comment]
Thanks. Is it possible to generate the output file for whatever is getting
logged in terminal?

Thank you.

**Comment by oarriaga** [comment]
There is a callback from keras that allows you to save the losses in a csv file. I believe the current training script contains such callback and is generating the log files while running.

**Comment by depakaakula** [comment]
I see the log is kinda unstructured. For the people who are not technical
enough to understand these numbers, is there a way to tune it a bit? Like when
and which each emotion is displayed.

Also, does each expression has unique value?

Thank you.

**Comment by depakaakula** [comment]
Dear oarriaga. 
The log file (fer2013_emotion_training.log) doesn't seem to be capturing my recent terminal report. Also, the output file data is kinda unstructured. Can you help me understand how to relate the expression and digits in the output file?

Thank you.



<img width="328" alt="screen shot 2017-11-30 at 00 21 22" src="https://user-images.githubusercontent.com/32492052/33415422-6e954034-d564-11e7-932b-e5226d40200b.png">
